src/serverenvironment.cpp

```diff
@@ -509,9 +509,9 @@ bool ServerEnvironment::line_of_sight(v3f pos1, v3f pos2, float stepsize, bool n
 
 		MapNode n = getMap().getNodeNoEx(pos);
 
-		if (n.drawtype != NDT_AIRLIKE) {
+		if (m_gamedef->ndef()->get(n).drawtype != NDT_AIRLIKE) {
```

Comment:
Travis was failing it before because I wasn't checking it properly, this seems the only wai to check .drawtype and .buildable_to without breaking code...  Am always up for suggestions and better ways to implement though :)))